android/app/src/main/java/org/uwaterloo/subletr/api/models/GetUserResponse.kt

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * Please note:
+ * This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * Do not edit this file manually.
+ *
+ */
+
+@file:Suppress(
+    "ArrayInDataClass",
+    "EnumEntryName",
+    "RemoveRedundantQualifierName",
+    "UnusedImport"
+)
+
+package org.uwaterloo.subletr.api.models
+
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * 
+ *
+ * @param firstName 
+ * @param lastName 
+ * @param email 
+ * @param gender 
+ */
+
+
+data class GetUserResponse (
```

Comment:
is it possible to add the listing that the user owns in the response? I'm thinking it would return a listing ID if they have one, and null if not

Reply:
Done!